components/builder-scheduler/src/server/scheduler.rs

```diff
@@ -426,8 +444,16 @@ impl ScheduleMgr {
                 }
 
                 match job.get_state() {
-                    jobsrv::JobState::Complete |
+                    jobsrv::JobState::Complete => {
+                        self.promote_package(
+                            &job.get_package_ident(),
+                            &bldr_channel_name!(job.get_owner_id()),
+                        )?;
```

Comment:
It might make it more explicit what's going on here to make a helper function like `promote_to_sandbox(&job)`